Makefile

```diff
@@ -141,6 +141,9 @@ run: build ## run the docker daemon in a container
 shell: build ## start a shell inside the build env
 	$(DOCKER_RUN_DOCKER) bash
 
+yamlfiles: build ## generate documentation YAML files consumed by docs repo
```

Comment:
Something like `yaml-docs-gen` would be great (much closer to the name of the bundle)